format-specs/geoparquet.md

```diff
@@ -58,15 +58,15 @@ Version of the GeoParquet spec used, currently 0.4.0
 
 Each geometry column in the dataset must be included in the columns field above with the following content, keyed by the column name:
 
-| Field Name | Type | Description |
-| --- | --- | --- |
-| encoding | string | **REQUIRED** Name of the geometry encoding format. Currently only `"WKB"` is supported. |
-| geometry_types | \[string] | **REQUIRED** The geometry types of all geometries, or an empty array if they are not known. |
-| crs | object | **OPTIONAL** [PROJJSON](https://proj.org/specifications/projjson.html) object representing the Coordinate Reference System (CRS) of the geometry. If the crs field is not included then the data in this column must be stored in longitude, latitude based on the WGS84 datum, and CRS-aware implementations should assume a default value of [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84). |
-| orientation | string | **OPTIONAL** Winding order of exterior ring of polygons. If present must be `"counterclockwise"`; interior rings are wound in opposite order. If absent, no assertions are made regarding the winding order. |
-| edges | string | **OPTIONAL** Name of the coordinate system for the edges. Must be one of `"planar"` or `"spherical"`. The default value is `"planar"`. |
-| bbox | \[number] | **OPTIONAL** Bounding Box of the geometries in the file, formatted according to [RFC 7946, section 5](https://tools.ietf.org/html/rfc7946#section-5). |
-| epoch | number | **OPTIONAL** Coordinate epoch in case of a dynamic CRS, expressed as a decimal year. |
+| Field Name     | Type         | Description |
+| -------------- | ------------ | ----------- |
+| encoding       | string       | **REQUIRED** Name of the geometry encoding format. Currently only `"WKB"` is supported. |
+| geometry_types | \[string]    | **REQUIRED** The geometry types of all geometries, or an empty array if they are not known. |
+| crs            | object\|null | **OPTIONAL** [PROJJSON](https://proj.org/specifications/projjson.html) object representing the Coordinate Reference System (CRS) of the geometry. If the crs field is not included then the data in this column must be stored in longitude, latitude based on the WGS84 datum, and CRS-aware implementations should assume a default value of [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84). |
```

Comment:
It is stated below, but since `null` is being added here, I think the text should also describe what `null` means.  Something like "a value of `null` indicates that the CRS is undefined or unknown."

Reply:
```suggestion
| crs            | object\|null | **OPTIONAL** [PROJJSON](https://proj.org/specifications/projjson.html) object representing the Coordinate Reference System (CRS) of the geometry. If the crs field is not included then the data in this column must be stored in longitude, latitude based on the WGS84 datum, and CRS-aware implementations should assume a default value of [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84). A value of `null` indicates that the CRS is undefined or unknown. |
```